index.html

```diff
@@ -4,24 +4,94 @@
 <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title></title>
-    <link rel=“stylesheet” href=“https://use.fontawesome.com/releases/v5.5.0/css/all.css”>
+    <title>Apple clone</title>
     <link
-        href="https://fonts.googleapis.com/css2?family=Montserrat:wght@900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+        href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,200;0,300;0,400;0,500;0,600;1,500;1,600&display=swap"
         rel="stylesheet" />
     <link
         href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,600;0,700;1,200;1,300;1,400;1,600;1,700&display=swap"
         rel="stylesheet" />
-
-    <link href="favicon.icon" rel="icon" type="image/x-icon" />
     <script src="https://kit.fontawesome.com/a076d05399.js"></script>
-
     <link rel="stylesheet" href="styles/style.css" />
 </head>
 
 <body>
-    <div>
-        <div>
+    <main class="grid-container">
+        <section class="main-section">
+            <div class="container">
+                <nav>
+                    <ul>
+                        <li>
+                            <a href=""><i class="fab fa-apple fa-2x"></i></a>
+                        </li>
+                        <li><a href="">Store</a></li>
+                        <li><a href="">Mac</a></li>
+                        <li><a href="">iPod</a></li>
+                        <li><a href="">iPhone</a></li>
+                        <li><a href="">iPad</a></li>
+                        <li><a href="">iTunes</a></li>
+                        <li><a href="">Support</a></li>
```

Comment:
- [ ] These eight elements should be links (to other sections or #) i.e. your `href` tags are empty.
Kindly check same for the rest of the page.